hat/assets/js/apps/Iaso/domains/home/HomeOnline.tsx

```diff
@@ -110,10 +120,23 @@ export const HomeOnline: FunctionComponent = () => {
                         >
                             <MenuIcon />
                         </IconButton>
-                        <SidebarMenuComponent location={window.location} />
+                        <SidebarMenuComponent
+                            location={window.location}
+                            activeLocale={{}}
+                        />
                     </Box>
                 </Grid>
                 <Grid container item xs={6} justifyContent="flex-end">
+                    <Box
+                        display="flex"
+                        alignItems="center"
+                        justifyContent="flex-end"
+                    >
+                        <CurrentUserInfos
+                            currentUser={currentUser}
+                            version={(window as any).IASO_VERSION}
```

Comment:
not sure for the cast here, but if this is working

Reply:
It works, this cast is about avoiding an error **Property 'IASO_VERSION' does not exist on type 'Window & typeof globalThis'.**